usaspending_api/api_contracts/contracts/v1/references/hash.md

```diff
@@ -12,6 +12,11 @@ Restore Filters From URL Hash Data
 + Request (application/json)
     + Attributes (object)
         + `hash` : `0e7d2ce3bb0885ac877872bb44053a84` (required, string)
+    + Schema
```

Comment:
Should this Schema block be added to all of the API contracts?

Reply:
As time allows; yes. We can do so incrementally.

Doing so allows the mock server to respond to requests for mock end points which do not match the request object defined in the contract.

For example w/ this endpoint:
- w/o the Schema definition, the mock server will only respond to service calls with `0e7d2ce3bb0885ac877872bb44053a84 ` in the request object.
- w/ the Schema definition, it will respond to anything that matches the schema more broadly; for example, `{ hash: 123 }`